<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIV</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>477</x>
      <y>1067</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
     <item>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="tabPosition">
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Exp Tree</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Capture</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>321</width>
           <height>165</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Search Adapters</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextBrowser" name="textBrowser_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Exposure</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="comboBox_6"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Open Adapter</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Capture</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_4"/>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_5"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>130</y>
           <width>321</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Timing Setup...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Power Setup...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>190</y>
           <width>321</width>
           <height>121</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Laser A</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Laser B</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string>Laser On</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>Laser Off</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Save Images</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string>Capture</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>310</y>
           <width>321</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Grab Stop</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>340</y>
           <width>321</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Save Setups</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_11"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_11">
            <property name="text">
             <string>Del</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_12">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Process</string>
        </attribute>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <attribute name="title">
         <string>Frame A</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Frame B</string>
        </attribute>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>631</width>
           <height>511</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Diplay</string>
        </attribute>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>641</width>
           <height>521</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExperiment_Run">
    <property name="title">
     <string>Experiment/Run</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuCapture">
    <property name="title">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuExperiment_Run"/>
   <addaction name="menuCapture"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Experiment"/>
   <addaction name="actionNew_Experiment_2"/>
   <addaction name="actionNew_Experiment_3"/>
   <addaction name="actionNew_Experiment_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Experiment_5"/>
   <addaction name="actionNew_Experiment_6"/>
   <addaction name="actionNew_Experiment_7"/>
   <addaction name="actionNew_Experiment_8"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionNew_Experiment_9"/>
   <addaction name="actionNew_Experiment_10"/>
   <addaction name="actionNew_Experiment_11"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Experiment_12"/>
   <addaction name="actionNew_Experiment_13"/>
   <addaction name="actionNew_Experiment_14"/>
   <addaction name="actionNew_Experiment_15"/>
  </widget>
  <action name="actionNew_Experiment">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/文档添加_doc-add.png</normaloff>:/PIV/文档添加_doc-add.png</iconset>
   </property>
   <property name="text">
    <string>New Experiment</string>
   </property>
   <property name="toolTip">
    <string>New Experiment</string>
   </property>
  </action>
  <action name="actionNew_Experiment_2">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/编辑_editor.png</normaloff>:/PIV/编辑_editor.png</iconset>
   </property>
   <property name="text">
    <string>Edit Experiment</string>
   </property>
   <property name="toolTip">
    <string>Edit Experiment</string>
   </property>
  </action>
  <action name="actionNew_Experiment_3">
   <property name="text">
    <string>New Run</string>
   </property>
   <property name="toolTip">
    <string>New Run</string>
   </property>
  </action>
  <action name="actionNew_Experiment_4">
   <property name="text">
    <string>Edit Run</string>
   </property>
   <property name="toolTip">
    <string>Edit Run</string>
   </property>
  </action>
  <action name="actionNew_Experiment_5">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/连接相机.png</normaloff>:/PIV/连接相机.png</iconset>
   </property>
   <property name="text">
    <string>Link Camera</string>
   </property>
   <property name="toolTip">
    <string>Link Camera</string>
   </property>
  </action>
  <action name="actionNew_Experiment_6">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/打开相机.png</normaloff>:/PIV/打开相机.png</iconset>
   </property>
   <property name="text">
    <string>Open Camera</string>
   </property>
   <property name="toolTip">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionNew_Experiment_7">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/相机设置.png</normaloff>:/PIV/相机设置.png</iconset>
   </property>
   <property name="text">
    <string>Set Camera</string>
   </property>
   <property name="toolTip">
    <string>Set Camera</string>
   </property>
  </action>
  <action name="actionNew_Experiment_8">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/关闭相机.png</normaloff>:/PIV/关闭相机.png</iconset>
   </property>
   <property name="text">
    <string>Close Camera</string>
   </property>
   <property name="toolTip">
    <string>Close Camera</string>
   </property>
  </action>
  <action name="actionNew_Experiment_9">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/放大_zoom-in.png</normaloff>:/PIV/放大_zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionNew_Experiment_10">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/缩小_zoom-out.png</normaloff>:/PIV/缩小_zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionNew_Experiment_11">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/自适应大小.png</normaloff>:/PIV/自适应大小.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Fit</string>
   </property>
   <property name="toolTip">
    <string>Zoom Fit</string>
   </property>
  </action>
  <action name="actionNew_Experiment_12">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/第一个.png</normaloff>:/PIV/第一个.png</iconset>
   </property>
   <property name="text">
    <string>First Image</string>
   </property>
   <property name="toolTip">
    <string>First Image</string>
   </property>
  </action>
  <action name="actionNew_Experiment_13">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/24gl-previousCircle.png</normaloff>:/PIV/24gl-previousCircle.png</iconset>
   </property>
   <property name="text">
    <string>Last Image</string>
   </property>
   <property name="toolTip">
    <string>Last Image</string>
   </property>
  </action>
  <action name="actionNew_Experiment_14">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/24gl-nextCircle.png</normaloff>:/PIV/24gl-nextCircle.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="toolTip">
    <string>Next Image</string>
   </property>
  </action>
  <action name="actionNew_Experiment_15">
   <property name="icon">
    <iconset resource="apprcc.qrc">
     <normaloff>:/PIV/最后一个.png</normaloff>:/PIV/最后一个.png</iconset>
   </property>
   <property name="text">
    <string>End Image</string>
   </property>
   <property name="toolTip">
    <string>End Image</string>
   </property>
  </action>
  <action name="actionNew_Experiment_16">
   <property name="text">
    <string>New Experiment</string>
   </property>
   <property name="toolTip">
    <string>New Experiment</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="apprcc.qrc"/>
 </resources>
 <connections/>
</ui>
